src/content/blog/community-day-2023.mdx

```diff
@@ -0,0 +1,188 @@
+---
+title: "Building a community of builders"
+description: "Today, we launch our newest initiative to recognize and celebrate people in our community who go above and beyond, including our very own maintainers."
+publishDate: "December 7, 2023"
+coverImage: "/src/content/blog/_images/community-day-2023/header.webp"
+socialImage: "/src/content/blog/_images/community-day-2023/og.webp"
+lang: "en"
+authors:
+  - sarah
+  - elian
+---
+
+import CommunityProject from '../../components/CommunityProject.astro'
+
+Astro continues to evolve how it works with and for its community. 
+
+With generous Open Collective funding support, we regularly recognize and reward people in our community who made a difference that month: submitting code and bug fixes, being welcoming and helpful in support threads, translating the docs, and other contributions that enrich everyone's Astro experience.
+
+Previously a quarterly event, Astro kicked it into high gear over the last few months and committed to issuing awards on a monthly basis. We also changed the nomination structure from peer-nominated awards to core-maintainer nominated awards. 
+
+This might sound less democratic, but our motivations were to **ensure that our core maintainers were actively on the lookout for the next crop of up-and-comers**. We now hold monthly maintainer meetings entirely dedicated to thinking about and shouting out who is most active in our community. As a core team, we dedicate time to our open-source community's health, growth, and sustainability.
+
+A community award is nice, but until now, this recognition has mostly stayed within our community. We wanted a better, more public, and more permanent place for community recognition. Somewhere you could show off to friends and family, maybe to explain where all those nights and weekends went!
+
+We are thrilled to announce the launch of our first public community space: https://community.astro.build built by our first Community Engineer, Elian van Cutsem.
+
+The first challenge was to design a friendly and welcoming homepage, which gives the visitor all the information about our Community Awards at a glance. For that, the DX team worked closely with our designer Mark to set up a homepage and design language. Once we were happy with the design, we started building an Astro website and translating our design into components, with a focus on accessibility and SEO.
+
+Right now, the website serves a single purpose: to showcase the Astro Community Award winners. But, our plan is to incrementally expand the website with more community initiatives & projects in the future.
+
+Today, you can visit and celebrate the people who help make Astro the project you love to use. Stay tuned for more exciting community initiatives that will find a home there as we grow and extend the site!
```

Comment:
Since the Community page will potentially include people that are no longer contributing, perhaps it makes sense to use "helped"?
```suggestion
Today, you can visit and celebrate the people who helped make Astro the project you love to use. Stay tuned for more exciting community initiatives that will find a home there as we grow and extend the site!
```